epoch/api/README.md

```diff
@@ -31,6 +31,13 @@ The epoch node exposes the following APIs:
     * It is defined in the rest of this document;
     * It is meant to be exposed on the Internet;
     * Its TCP port is configurable.
+  * Range of external Channels noise endpoints. It consists of as many TCP
```

Comment:
Which Noise public key is used by the listener?
Which Noise public key of the other party does the listener expect/validate?

Reply:
[Addressed](https://github.com/aeternity/protocol/pull/229/commits/a086114ae96dd9f500cbc6e03869d5ee3a2b0502#diff-8b16649bc0188ba5b35d5f92f6b8656aR40)